### PR TITLE
#165020192 Install eslint

### PR DESCRIPTION
**Pull request by fridolinho**
## What does this PR do?
configure eslint

## Description of Task to be completed?
- install eslint locall
- install airbnb style guide
- install Static AST checker for accessibility rules on JSX elements.
- install React specific linting rules for ESLint
- install es6 import/export linting rules
## How should this be manually tested?

   Run npm install command then initialise Eslint

## Any background context you want to provide?
N/A

## What are the relevant pivotal tracker stories?
[ch-165020192](https://www.pivotaltracker.com/story/show/165020192)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,13 +20,20 @@
...
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
     "request": "^2.87.0",
-    "slug": "^0.9.1",
+    "slug": "^1.1.0",
     "underscore": "^1.9.1"
+  },
+  "devDependencies": {
+    "eslint": "^5.16.0",
+    "eslint-config-airbnb": "^17.1.0",
```
Comment (niyobobo) [inline package.json:34]: I seed dependencies but no eslint Config file. You can check if you can push it also

Comment (fridolinho) [reply package.json:34]: Thank you, I pushed it.

Comment (niyobobo) [reply package.json:34]: Yeah, Now It's good!

**Review by daud1 — APPROVED** [review]
LGTM

**Pull request merged**